frontend/app/assets/stylesheets/spree/frontend/views/spree/orders/edit.scss

```diff
@@ -354,6 +359,11 @@
         }
       }
     }
+    input[type=text] {
```

Comment:
@PSzyszkaSpark it's not only text but also, email, password, number

Reply:
@damianlegawiec I've added email, password, number types to separate locations, where they needed to be placed. In cart there are only inputs with type string.